Sources/TokamakDemo/Counter.swift

```diff
@@ -27,16 +27,15 @@ public struct Counter: View {
   let limit: Int
 
   public var body: some View {
```

Comment:
Same here

Reply:
Turns out adding on the protocol is enough.

Reply:
Ah, yeah that is new but not included in beta 1 of Xcode 12. Is it in beta 2? I haven't gotten it installed yet..

Reply:
Yes, in beta 2 apparently, but already works with SwiftWasm snapshot. I'll keep the PR in draft state until GHA gets beta 2 for CI to pass.